bower.json

```diff
@@ -24,8 +24,8 @@
   ],
   "dependencies": {
     "jquery": "^1.8.0 || ^2.1.0",
-    "backbone" : "^1.1.0",
-    "underscore": "^1.6.0"
+    "backbone" : "^1.2.1",
```

Comment:
Sorry, this still has to be `^1.1.0` or `1.0.0-1.2.1` (same with package.json)
/cc @samccone